google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -86,16 +89,19 @@ public boolean isModified() {
   }
 
   /**
-   * User's shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
+   * Users shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
    */
   @Override
   public void apply() throws ConfigurationException {
-    if (StringUtil.isEmpty(cloudSdkPanel.getCloudSdkDirectory())
-        || !CloudSdkUtil.containsCloudSdkExecutable(cloudSdkPanel.getCloudSdkDirectory())) {
+    try {
+      new CloudSdk.Builder()
```

Comment:
Yes, I discussed this with Patrick two weeks ago. If we're moving to a strategy of validating the Cloud SDK per action/method, then we'll probably not validate here anymore. This needs some more thought, but I don't think fixing this should be in the scope of this PR.